frontend/src/components/TreeViewer/TreeViewer.jsx

```diff
@@ -305,7 +377,6 @@ export const TreeViewer = ({ data }) => {
       clickedNode.data.name,
       curViewingNodeID.current
     );
-    //previouslyClickedNode.current = clickedNode.data.name;
     const nodeID = getParameterByName("id", clickedNode.data.url);
```

Comment:
There already exist functions in JS for doing this: https://www.sitepoint.com/get-url-parameters-with-javascript/. I think it's safer to use this instead.

Reply:
Seems like this is out of scope for this MR; instead we can address it in the other cleanup MR here: https://github.com/VerWiki/VerWikiApp/pull/21